tensorflow/python/kernel_tests/lookup_ops_test.py

```diff
@@ -3309,6 +3309,75 @@ def testMutableHashTableFindHighRank(self):
       result = self.evaluate(output)
       self.assertAllEqual([[0, 1], [-1, -1]], result)
 
+  def testMutableHashTableFindWithInvalidShapeDefaultValue(self):
+    with self.cached_session():
```

Comment:
I think you can remove self.cached_session()

Reply:
Accept